### PR TITLE
Atlassian Statuspage Ocean Integration Docs

### PR DESCRIPTION
**Pull request by phalbert**
# Description

This PR adds comprehensive documentation for integrating [Atlassian Statuspage with Ocean](https://github.com/port-labs/ocean/commit/de6057cc6673092b1664cac672755017d1df8cb6)

## Added docs pages

Please also include the path for the added docs

- Statuspage (`/docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md`)
- Docker Parameters (`/docs/build-your-software-catalog/sync-data-to-catalog/incident-management/_statuspage_docker_parameters.mdx`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 1]
```diff
@@ -0,0 +1,1750 @@
...
+  "status": "resolved",
+  "updated_at": "2024-06-07T22:26:34Z",
+  "reminder_intervals": null
+}
+```
+
+</details>
+
+<details>
+<summary> Incident Update response data</summary>
```
Comment (lordsarcastic) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:1556]:
```suggestion
<summary><b>Incident Update response data (Click to expand)</b></summary>
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 2]
```diff
@@ -0,0 +1,1750 @@
...
+}
+```
+</details>
+
+### Mapping Result
+
+The combination of the sample payload and the Ocean configuration generates the following Port entity:
+
+<details>
+<summary> Page entity in Port</summary>
```
Comment (lordsarcastic) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:1582]:
```suggestion
<summary><b>Page entity in Port (Click to expand)</b></summary>
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 3]
```diff
@@ -0,0 +1,1750 @@
...
+  "createdBy": "<port-client-id>",
+  "updatedAt": "2024-06-25T07:59:28.716Z",
+  "updatedBy": "<port-client-id>"
+}
+```
+
+</details>
+
+<details>
+<summary>Component Group entity in Port</summary>
```
Comment (lordsarcastic) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:1615]:
```suggestion
<summary><b>Component Group entity in Port(Click to expand)</b></summary>
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 4]
```diff
@@ -0,0 +1,1750 @@
...
+  "createdAt": "2024-06-25T07:59:33.363Z",
+  "createdBy": "<port-client-id>",
+  "updatedAt": "2024-06-25T07:59:33.363Z",
+  "updatedBy": "<port-client-id>"
+}
+```
+</details>
+
+<details>
+<summary> Component entity in Port</summary>
```
Comment (lordsarcastic) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:1641]:
```suggestion
<summary><b>Component entity in Port (Click to expand)</b></summary>
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 5]
```diff
@@ -0,0 +1,1750 @@
...
+  "createdAt": "2024-06-25T07:59:35.830Z",
+  "createdBy": "<port-client-id>",
+  "updatedAt": "2024-06-25T07:59:35.830Z",
+  "updatedBy": "<port-client-id>"
+}
+```
+</details>
+
+<details>
+<summary> Incident entity in Port</summary>
```
Comment (lordsarcastic) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:1672]:
```suggestion
<summary><b>Incident entity in Port (Click to respond)</b></summary>
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 6]
```diff
@@ -0,0 +1,1750 @@
...
+  "createdBy": "<port-client-id>",
+  "updatedAt": "2024-06-25T08:00:39.222Z",
+  "updatedBy": "<port-client-id>"
+}
+```
+</details>
+
+
+<details>
+<summary> Incident Update entity in Port</summary>
```
Comment (lordsarcastic) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:1722]:
```suggestion
<summary><b>Incident Update entity in Port (Click to expand)</b></summary>
```

**Review by lordsarcastic — CHANGES_REQUESTED** [review]
A few changes

**Review by lordsarcastic — APPROVED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 7]
```diff
@@ -0,0 +1,1750 @@
...
+import Prerequisites from "../templates/\_ocean_helm_prerequisites_block.mdx"
+import AzurePremise from "../templates/\_ocean_azure_premise.mdx"
+import DockerParameters from "./\_statuspage_docker_parameters.mdx"
+import HelmParameters from "../templates/\_ocean-advanced-parameters-helm.mdx"
+import AdvancedConfig from '../../../generalTemplates/_ocean_advanced_configuration_note.md'
+import PortApiRegionTip from "/docs/generalTemplates/_port_region_parameter_explanation_template.md"
+
+# Statuspage
+
+Our [Atlassian Statuspage](https://www.atlassian.com/software/statuspage) integration allows you to import `page`, `component_group`, `component`, `incident`, and `incident_update` resources from your Statuspage instance into Port, according to your mapping and definitions.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:16]:
```suggestion
Port's [Atlassian Statuspage](https://www.atlassian.com/software/statuspage) integration allows you to import `page`, `component_group`, `component`, `incident`, and `incident_update` resources from your Statuspage instance into Port, according to your mapping and definitions.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 8]
```diff
@@ -0,0 +1,1750 @@
...
+import TabItem from "@theme/TabItem"
+import Prerequisites from "../templates/\_ocean_helm_prerequisites_block.mdx"
+import AzurePremise from "../templates/\_ocean_azure_premise.mdx"
+import DockerParameters from "./\_statuspage_docker_parameters.mdx"
+import HelmParameters from "../templates/\_ocean-advanced-parameters-helm.mdx"
+import AdvancedConfig from '../../../generalTemplates/_ocean_advanced_configuration_note.md'
+import PortApiRegionTip from "/docs/generalTemplates/_port_region_parameter_explanation_template.md"
+
+# Statuspage
+
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:15]: Refrain from using "our" in cases like these, when talking about something of Port's use "Port's".

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 9]
```diff
@@ -0,0 +1,1750 @@
...
+  :::tip Blueprint key
+  Note the value of the `blueprint` key - if you want to use a hardcoded string, you need to encapsulate it in 2 sets of quotes, for example use a pair of single-quotes (`'`) and then another pair of double-quotes (`"`)
+  :::
+
+### Ingest data into Port
+
+To ingest Statuspage objects using the [integration configuration](#configuration-structure), you can follow the steps below:
+
+1. Go to the DevPortal Builder page.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:450]: should be:
Go to the [Builder page](link to builder) of your portal.

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 10]
```diff
@@ -0,0 +1,1750 @@
...
+2. Select the Data Sources tab at the left sidebar.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:451]:
```suggestion
2. Select the `Data Sources` tab in the left sidebar.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 11]
```diff
@@ -0,0 +1,1750 @@
...
+3. Click on `+ Data Source` at the top right corner.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:452]:
```suggestion
3. Click on `+ Data Source` in the top right corner.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 12]
```diff
@@ -0,0 +1,1750 @@
...
+4. Select Statuspage under the Incident Management category.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:453]:
```suggestion
4. Select `Statuspage` under the `Incident Management` category.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 13]
```diff
@@ -0,0 +1,1750 @@
...
+5. Modify the [configuration](#configuration-structure) according to your needs.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:454]:
```suggestion
5. Modify the [configuration](#configuration-structure) according to your data.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 14]
```diff
@@ -0,0 +1,1750 @@
...
+          mappings: ...
+  ```
+
+  :::tip Blueprint key
+  Note the value of the `blueprint` key - if you want to use a hardcoded string, you need to encapsulate it in 2 sets of quotes, for example use a pair of single-quotes (`'`) and then another pair of double-quotes (`"`)
+  :::
+
+### Ingest data into Port
+
+To ingest Statuspage objects using the [integration configuration](#configuration-structure), you can follow the steps below:
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:448]:
```suggestion
To ingest Statuspage objects using the [integration configuration](#configuration-structure), follow the steps below:
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 15]
```diff
@@ -0,0 +1,1750 @@
...
+
+1. Go to the DevPortal Builder page.
+2. Select the Data Sources tab at the left sidebar.
+3. Click on `+ Data Source` at the top right corner.
+4. Select Statuspage under the Incident Management category.
+5. Modify the [configuration](#configuration-structure) according to your needs.
+6. Run the installation command.
+7. Click `Next` and you can view the integration configuration and update it as necessary.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:456]:
```suggestion
7. Click `Next`, you can view the integration configuration and update it as necessary.
```

**Review by hadar-co — CHANGES_REQUESTED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md [hunk 16]
```diff
@@ -0,0 +1,1750 @@
...
+  Note the value of the `blueprint` key - if you want to use a hardcoded string, you need to encapsulate it in 2 sets of quotes, for example use a pair of single-quotes (`'`) and then another pair of double-quotes (`"`)
+  :::
+
+### Ingest data into Port
+
+To ingest Statuspage objects using the [integration configuration](#configuration-structure), follow the steps below:
+
+1. Go to the [Builder page](https://app.getport.io/dev-portal/data-model) of your portal.
+2. Select the `Data Sources` tab in the left sidebar.
+3. Click on `+ Data Source` button in the top right corner.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/statuspage.md:452]:
```suggestion
3. Click on the `+ Data Source` button in the top right corner.
```

**Review by hadar-co — APPROVED** [review]

**Pull request merged**